com.unity.netcode.gameobjects/Editor/CodeGen/com.unity.netcode.editor.codegen.asmdef

```diff
@@ -15,5 +15,12 @@
         "Mono.Cecil.Pdb.dll",
         "Mono.Cecil.Rocks.dll"
     ],
-    "autoReferenced": false
-}
\ No newline at end of file
+    "autoReferenced": false,
+    "versionDefines": [
+        {
+            "name": "Unity",
+            "expression": "(0,2022.2.0a11)",
+            "define": "UNITY_CECIL_CONSTRAINTS_ARE_TYPE_REFERENCES"
+        }
+    ]
```

Comment:
I checked our internal thread on Slack and looks like the update is coming from `com.unity.nuget.mono-cecil` package, not the Engine directly. I guess we just updated Editor manifest file such that it points to a newer version of `com.unity.nuget.mono-cecil` from 2022.2.0a11 and onwards. if so, shouldn't we treat this as a package dep upgrade rather than engine upgrade?

Reply:
Is there a reason why we target a package from an alpha release in the first place? I would think it'd be safest to use whatever was in the last stable version (21 LTS), and then after the next final stable version comes out, we upgrade to that (and in this case implement the if/else)